packages/angular/README.md

```diff
@@ -1,32 +1,31 @@
-# Installation and Code Generation
+# @coveo/angular
 
-The `@coveo/angular` package contains the necessary components to set up a search page with Angular CLI.
+The `@coveo/angular` package contains the necessary components to set up a search page using [Coveo Headless](https://docs.coveo.com/headless) and [Angular Material](https://material.angular.io/) with the [Angular CLI](https://cli.angular.io/).
 
-## Install Schematic
+If you want to create a new Coveo Headless-powered search page with the Angular web framework from scratch, it is simpler to use the [`coveo ui:create:angular` command in the main Coveo CLI](https://github.com/coveo/cli/blob/master/packages/cli/README.md#coveo-uicreateangular-name). The main Coveo CLI will handle a lot of complexity for you. The `@coveo/angular` package is rather meant to help you integrate a search page to an already existing Angular project using the Angular CLI.
 
-Using the command below will set up a Coveo search page in your project
+## Installation
 
-```
-ng add @coveo/angular --org-id<org-id> --api-key=<api-key>
-```
+To add a Coveo Headless-powered search page using Material Angular to an existing Angular project, execute the following steps:
+
+- Run `ng add @coveo/angular --org-id=<org-id> --api-key=<api-key>`, where you replace `<org-id>` by the unique identifier of your Coveo organization, and `<api-key>` by an API key granting the **impersonate** privilege in the target organization.
+- Answer the questions prompted by the shell script to configure your interface styling.
+- In your project, navigate to the new `server` folder. This is an [Express](https://www.npmjs.com/package/express) server which generates [Coveo search tokens](https://docs.coveo.com/en/1346/).
+- The folder should contain a `.env.example` file. Create a copy of that file and rename it `.env`.
+- In that new `.env` file, replace all placeholder variables (`<...>`) by the proper information for your organization. For more involved configurations, you can modify the request parameters used in the `middlewares/searchToken.ts` file.
+- In that server directory, run `npm install` to install dependencies.
+- Still in that server directory, run `npm start` to start the server.
```

Comment:
the `npm start` should be executed at the workspace root.

Reply:
hmmm... I was starting the server in its own because otherwise I had this:
<img width="801" alt="Screen Shot 2021-03-24 at 12 10 14 PM" src="https://user-images.githubusercontent.com/39384459/112343857-e7191e00-8c99-11eb-9f30-9d545765e1c8.png">
Seems like there's no /scripts directory at all 🤔 
<img width="307" alt="Screen Shot 2021-03-24 at 12 12 10 PM" src="https://user-images.githubusercontent.com/39384459/112344127-2ba4b980-8c9a-11eb-9d76-454015177c36.png">

Let me know if you'd like to have a call to discuss this. I have a meeting between 1pm-2:30pm but otherwise I'd be available.

Reply:
Oh is seems like the angular schematic is being published without built first.

As you can see the source code has the missing file

![image](https://user-images.githubusercontent.com/12199712/112377887-a1ba1800-8cbc-11eb-9656-71682e652769.png)


But the dist folder (which is used when you call the schematic) doesn't have it. I'll open a bug
![image](https://user-images.githubusercontent.com/12199712/112377921-abdc1680-8cbc-11eb-9907-661b60d3908a.png)

Reply:
I see 😅 
I'll just rewrite as if it was working then 🙃